File: backend/Kanban.Core/Entities/Task.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Kanban.Core.Entities
{
    public  class Task
    {
        public int Id { get; set; }
        public string Title { get; set; } = string.Empty;
        public string? Description { get; set; }
        public TaskStatus Status { get; set; } = TaskStatus.ToDo;
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
        public DateTime UpdatedAt { get; set; } = DateTime.UtcNow;
    }
}

File: frontend/src/types/task.ts
export enum KanbanTaskStatus {
  TODO = "ToDo",
  INPROGRESS = "InProgress",
  DONE = "Done",
}

export interface KanbanTask {
  id: number;
  title: string;
  description?: string;
  status: KanbanTaskStatus;
  createdAt: string;
  updatedAt: string;
}
